packages/utils/src/requestdata.ts

```diff
@@ -264,9 +270,12 @@ export function addRequestDataToEvent(
   };
 
   if (include.request) {
-    const extractedRequestData = Array.isArray(include.request)
-      ? extractRequestData(req, { include: include.request })
-      : extractRequestData(req);
+    const includeRequest = Array.isArray(include.request) ? [...include.request] : [...DEFAULT_REQUEST_INCLUDES];
```

Comment:
q: do we need to spread `include.request` here?

Reply:
yeah, because we mutate this below possibly (adding the ip), and don't want to mutate what the user passes in! (ran into this in a test...)